lib/ajv.ts

```diff
@@ -31,39 +34,65 @@ class Ajv extends AjvCore {
   }
 }
 
-module.exports = exports = Ajv
+export = Ajv
 Object.defineProperty(exports, "__esModule", {value: true})
```

Comment:
```suggestion
Object.defineProperty(Ajv, "__esModule", {value: true})
```

The `export =` line causes a `module.exports = Ajv` to be written at the end of the file, so these mutations of the free variable `exports` aren’t going anywhere anymore. I missed this in my original suggestion.